### ISSUE TITLE
Suggestion(s)

### ISSUE THREAD

**Issue by nkigen** [JB-RESEARCHLABS/Online_Construction-APP#1]
I think it would be better to cover a much wider field of "projects" instead of just construction project since this idea can be applied to all kinds of projects.

**Comment by briankasozi** [comment]
Yes i think that's a nice idea but why don't we build it in modules ie. If we have a this module, we could easily integrate any other module because different projects have different requirements.
So when we decide on how to do this, i should start on the requirement engineering but having other projects actually sounds a nice idea because there we aren't fixed on one thing
